UnitTests/Extensions/Test_FontIconExtensionMarkupExtension.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Uwp.UI.Extensions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.AppContainer;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Markup;
+
+namespace UnitTests.Extensions
+{
+    [TestClass]
+    public class Test_FontIconExtensionMarkupExtension
+    {
+        [TestCategory("FontIconExtensionMarkupExtension")]
+        [UITestMethod]
+        public void Test_FontIconExtension_MarkupExtension_ProvideSegoeMdl2Asset()
+        {
+            var treeroot = XamlReader.Load(@"<Page
+    xmlns=""http://schemas.microsoft.com/winfx/2006/xaml/presentation""
+    xmlns:x=""http://schemas.microsoft.com/winfx/2006/xaml""
+    xmlns:ex=""using:Microsoft.Toolkit.Uwp.UI.Extensions"">
+        <AppBarButton x:Name=""Check"" Icon=""{ex:FontIcon Glyph=&#xE14D;}""/>
```

Comment:
```suggestion
        <AppBarButton x:Name=""Check"" Icon=""{ex:FontIcon Glyph=&#xE105;}""/>
```
You accidently weren't checking for the same char! 🤦‍♂️ 😋

Reply:
Whoops, thanks! 😄